src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -389,8 +389,8 @@ def apply_filesystem(self, spec):
         # TODO use spec.placement
         # TODO warn if spec.extended has entries we don't kow how
         #      to action.
-        def _update_fs(current, new):
-            # type: (cfs.CephFilesystem, cfs.CephFilesystem) -> cfs.CephFilesystem
+        def _update_fs(new):
+            # type: (cfs.CephFilesystem) -> cfs.CephFilesystem
```

Comment:
```suggestion
        def _update_fs(new: cfs.CephFilesystem) -> cfs.CephFilesystem:
```

converting these to py3 style ?

Reply:
Done